husky_components/matrix/main.js

```diff
@@ -203,9 +205,9 @@ define(function() {
             this.sandbox.dom.data($tdHead, 'section', this.options.values.vertical[rowCount]);
             this.sandbox.dom.append($tr, $tdHead);
 
-            // flag for checking if every flag is true
+            // flag for checking if every flag is false
             // insert values of matrix
-            allActive = this.prepareTableColumn(
+            allInactive = this.prepareTableColumn(
```

Comment:
Or rename it to `someActive`? Would be the actual positive variant of this name.